.circleci/config.yml

```diff
@@ -145,7 +145,7 @@ jobs:
           at: ~/
       - run:
           name: Build dist/
-          command: npm run build
```

Comment:
Can we put this in a separate `- run` block with its own name?

Reply:
Good call. Done in https://github.com/plotly/plotly.js/pull/5383/commits/4c7ad71f1fc04c643639434774c424ebe3b8c811.